curator/run_cron.py

```diff
@@ -27,16 +27,23 @@
 
 filename = os.getenv('CURATOR_CONF_LOCATION', '/etc/curator') + '/settings'
 
-decoded = []
+decoded = {}
 with open(filename, 'r') as stream:
-    decoded = yaml.load(stream) or []
+    decoded = yaml.load(stream) or {}
 
 connection_info = '--host ' + os.getenv('ES_HOST') + ' --port ' + os.getenv('ES_PORT') + ' --use_ssl --certificate ' + os.getenv('ES_CA') + ' --client-cert ' + os.getenv('ES_CLIENT_CERT') + ' --client-key ' + os.getenv('ES_CLIENT_KEY')
 
+defaults = {'.defaults': {'delete': {'days': int(os.getenv('CURATOR_DEFAULT_DAYS', 30))}}}
+
+default_time_unit = decoded.get('.defaults', defaults).get('delete', defaults['.defaults']['delete']).keys()[0]
+default_value = int(decoded.get('.defaults', defaults).get('delete', defaults['.defaults']['delete'])[default_time_unit])
+
 base_default_cmd = '/usr/bin/curator --loglevel ERROR ' + connection_info + ' delete indices --timestring %Y.%m.%d'
-default_command = base_default_cmd + ' --older-than ' + os.getenv('DEFAULT_DAYS') + ' --time-unit days' + ' --exclude .searchguard*' + ' --exclude .kibana*'
+default_command = base_default_cmd + ' --older-than ' + str(default_value) + ' --time-unit ' + default_time_unit + ' --exclude .searchguard*' + ' --exclude .kibana*'
```

Comment:
The time values are "validated" by the `int()` function::

    >python
    >>> int("a")
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    ValueError: invalid literal for int() with base 10: 'a'
    >>> 

So there will be a traceback in the curator logs.